python/fledge/services/core/user_model.py

```diff
@@ -172,13 +180,14 @@ async def update(cls, user_id, user_data):
                 result = await storage_client.update_tbl("users", payload)
                 if result['rows_affected']:
                     # FIXME: FOGL-1226 active session delete only in case of role_id and password updation
-
-                    # delete all active sessions
-                    await cls.delete_user_tokens(user_id)
+                    if 'password' in user_data or 'role_id' in user_data:
```

Comment:
also if access method is chnaged from any/cert/pwd to pwd/cert itself (not to any)